scripts/github/update-gh-pages.sh

```diff
@@ -22,11 +22,59 @@ create_versions_js() {
     # 'stable' is a symlink pointing to the latest version
     [ -f stable ] && echo "    { name: 'stable', url: '$_baseurl/stable' },"
     for f in `ls -d */  | tr -d /` ; do
-        echo "    { name: '$f', url: '$_baseurl/$f' },"
+        if [ -f "$f/index.html" ]; then
+            echo "    { name: '$f', url: '$_baseurl/$f' },"
+        fi
     done
     echo -e "  ];\n}"
 }
 
+# Helper for updating help repo index
+update_helm_repo_index() {
+    echo "Updating Helm repo index"
+
+    # TODO: with a lot of releases github API will paginate and this will break
+    releases="`curl -sSf -H 'Accept: application/vnd.github.v3+json' \
+        $GITHUB_API_URL/repos/$GITHUB_REPOSITORY/releases | jq -c '.[]'`"
+
+    for release_meta in $releases; do
+        # Set fields we're interested in as shell variables
+        eval `echo $release_meta | jq -r '{tag_name, url, assets} | keys[] as $k | "\($k)='"'"'\(.[$k])'"'"'"'`
+
+        for asset_meta in `echo $assets | jq -c '.[]'`; do
+            # Set fields we're interested in as "asset_<field>" shell variables
+            eval `echo $asset_meta | jq -r '{id, name, url, browser_download_url} | keys[] as $k | "local asset_\($k)=\(.[$k])"'`
+
+            if [[ "$asset_name" != node-feature-discovery-chart-*tgz ]]; then
+                echo "Asset $asset_name does not look like a Helm chart archive, skipping..."
+                continue
+            fi
+
+            # Check if the asset has changed
+            asset_id_old=`cat "$asset_name".id 2> /dev/null || :`
```

Comment:
do you expect changes to existing assets ?

im wondering if its possible to just query draft releases and update index.yaml for those only.

Reply:
Uploading of the assets is a manual process so I would keep this as a way correct mistakes, e.g. the maintainer fatfingering something